src/Morphir/TypeScript/Backend.elm

```diff
@@ -238,3 +307,15 @@ mapTypeExp tpe =
 
         Type.Function a argType returnType ->
             TS.UnhandledType "Function"
+
+
+{-| Utility funciton: map an AccessControlled.Access object, to a TS.Privacy Object
+-}
```

Comment:
This function has a clear name and a clear type annotation, so the comment above isn't needed.

Reply:
I'm not clear one what the best practice is with these comments. I'm pretty sure that if they're formatted right, they end up in the documentation, and I'd generally rather have a comment than no comment.

Then again, I don't know if this is a function that ought to be exported, so maybe it shouldn't even show up in the documentation anyway?

Either way, happy to remove if you're sure.

Reply:
> This function has a clear name and a clear type annotation

Also: Thanks!

I've decided to take that as a complement on writing clear comprehensible code.  :smile:

Reply:
Ah, documentation is a good point - but this function internal-only.